src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -37,16 +37,21 @@ interface BuildLocator {
      * By default only successful builds are returned, call this method to include failed builds as well.
      */
     fun withAnyStatus() : BuildLocator
-
     fun withStatus(status: BuildStatus): BuildLocator
+
     fun withTag(tag: String): BuildLocator
 
-    fun withBranch(branch: String): BuildLocator
+    /**
+     * By default failed to start builds are filtered out, call this method to include them.
+     */
+    fun withAnyFailedToStart(): BuildLocator
+    fun withFailedToStart(value: Boolean): BuildLocator
```

Comment:
Ok, will rename methods to 'includeFailedToStart()' and 'failedToStartOnly()', thank you! By default failed-to-start builds are ignored.

As a side note, I think there should always be a method to revert an option to default.